### DISCUSSION TITLE
Word Count

### DISCUSSION THREAD

**Discussion by micahvandegrift** [knowledgefutures/pubpub#1838] [General]
Is there a way to generate a word count from a Book?

**Comment by gabestein** [comment]
This just came up on the Pub level in a dev discussion, and I think it's something we'll prioritize relatively highly. Don't think it would be hard to add that total to the collection level once we have it!

**Reply by micahvandegrift** [reply]
Thanks Gabe. The ask came from a project partner who is taking the PubPub book thing to a publisher to become a IRL book thing and, you know, word count matters when we're killing trees to print! ;)
